### ISSUE TITLE
Documentation for ShardingSphere Agent may can simplify things

### ISSUE THREAD

**Issue by linghengqian** [apache/shardingsphere#20936]
## Question

**For English only**, other languages will not accept.

Before asking a question, make sure you have:

- Googled your question.
- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

- In https://shardingsphere.apache.org/document/5.2.0/en/user-manual/shardingsphere-proxy/observability/#usage-in-shardingsphere-proxy , it mentions that `Configure the absolute path of shardingsphere-agent.jar to the start.sh startup script of shardingsphere proxy.` is required.
- As far as I understand, modifying `start.sh` is very tedious work, and javaagent can be configured in the environment variable `JVM_OPTS`. This should be similar to `-javaagent /xxxxx/agent/shardingsphere-agent.jar`. As mentioned in https://github.com/cloudfoundry/java-buildpack/issues/555#issuecomment-368583041, requires modification `start.sh` complicates things. Refer to https://shardingsphere.apache.org/document/5.2.0/en/user-manual/shardingsphere-proxy/startup/docker/ .
- ![image](https://user-images.githubusercontent.com/20187731/189606435-c98c163b-103d-4abb-a55b-7941fc2911c2.png)

**Comment by RaigorJiang** [comment]
Yes, modifying `start.sh` is a bit fiddly, especially in a container environment.
We should consider a way to quickly control whether to start the agent or not.

**Comment by linghengqian** [comment]
- How should I reopen this issue?  I'm just waiting for one of my PRs to be merged...

**Comment by RaigorJiang** [comment]
> * How should I reopen this issue?  I'm just waiting for one of my PRs to be merged...

Hi @linghengqian , which PR is related to this issue?

**Comment by linghengqian** [comment]
- Actually I was talking about https://github.com/apache/shardingsphere/pull/21341 ,🤣 but due to recent changes in master branch, it had a merge conflict not long ago, which has now been resolved.
- Since I don't really want to manage multiple PRs at the same time, I usually don't work on issues in parallel. For this current issue, I will submit a PR shortly.
